repair/repair.cc

```diff
@@ -455,8 +458,11 @@ bool repair_module::is_aborted(const tasks::task_id& uuid) {
 void repair_module::abort_all_repairs() {
     _aborted_pending_repairs = _pending_repairs;
     for (auto& x : _repairs) {
-        auto& ri = x.second;
-        ri->abort();
+        auto it = _tasks.find(x.second);
+        if (it != _tasks.end()) {
+            auto& impl = dynamic_cast<shard_repair_task_impl&>(*it->second->_impl);
+            impl.abort_repair_info();
```

Comment:
I guess we can't use a generic repair method for this yet
but this is addressed in a follow up series.
If so, please add a respective FIXME comment